modules/coin/rest/handlers.go

```diff
@@ -0,0 +1,119 @@
+package rest
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/gorilla/mux"
+
+	"github.com/tendermint/basecoin"
+	"github.com/tendermint/basecoin/client/commands"
+	"github.com/tendermint/basecoin/client/commands/proofs"
+	"github.com/tendermint/basecoin/modules/auth"
+	"github.com/tendermint/basecoin/modules/base"
+	"github.com/tendermint/basecoin/modules/coin"
+	"github.com/tendermint/basecoin/modules/fee"
+	"github.com/tendermint/basecoin/modules/nonce"
+	"github.com/tendermint/basecoin/stack"
+	lightclient "github.com/tendermint/light-client"
+	"github.com/tendermint/tmlibs/common"
+)
+
+// SendInput is the request to send an amount from one actor to another.
+// Note: Not using the `validator:""` tags here because SendInput has
+// many fields so it would be nice to figure out all the invalid
+// inputs and report them back to the caller, in one shot.
+type SendInput struct {
+	Fees     *coin.Coin `json:"fees"`
+	Multi    bool       `json:"multi,omitempty"`
+	Sequence uint32     `json:"sequence"`
+
+	To     *basecoin.Actor `json:"to"`
+	From   *basecoin.Actor `json:"from"`
+	Amount coin.Coins      `json:"amount"`
+}
+
+func RegisterHandlers(r *mux.Router) error {
+	r.HandleFunc("/build/send", doSend).Methods("POST")
+	r.HandleFunc("/query/account/{signature}", doQueryAccount).Methods("GET")
+	return nil
+}
+
+// doQueryAccount is the HTTP handlerfunc to query an account
+// It expects a query string with
+func doQueryAccount(w http.ResponseWriter, r *http.Request) {
+	query := mux.Vars(r)
+	signature := query["signature"]
+	actor, err := commands.ParseActor(signature)
+	if err != nil {
+		common.WriteError(w, err)
+		return
+	}
+	actor = coin.ChainAddr(actor)
+	key := stack.PrefixedKey(coin.NameCoin, actor.Bytes())
+	account := new(coin.Account)
+	proof, err := proofs.GetAndParseAppProof(key, account)
+	if lightclient.IsNoDataErr(err) {
+		err := fmt.Errorf("account bytes are empty for address: %q", signature)
+		common.WriteError(w, err)
+		return
+	} else if err != nil {
+		common.WriteError(w, err)
+		return
+	}
+
+	if err := proofs.OutputProof(account, proof.BlockHeight()); err != nil {
```

Comment:
So that function looks like this 
<img width="773" alt="screen shot 2017-08-01 at 11 04 56 pm" src="https://user-images.githubusercontent.com/4898263/28858477-007e045a-770e-11e7-8195-464381275f23.png">
and doesn't expose `proof`, I'll duplicate it then, just letting you know what's up with it.